.github/workflows/ci.yaml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [main]
     paths-ignore:
-      - "**.md"
       - "**.yml"
       - "**.yaml"
```

Comment:
While we are at it, why not remove the `yaml` filter? Those files are for workflows, and it would seem logical to me that a change in a workflow should trigger the actions.

Reply:
I agree. Let's remove those constraints and see how the CI behaves with that.